api/wrappers.js

```diff
@@ -264,7 +335,7 @@ function wrapClient(cls) {
   Decorate the Pinger facade class.
 
   @param {Object} cls The auto-generated class.
-  @return {Object} The decorated class.
+  @returns {Object} The decorated class.
```

Comment:
technically both are supported....but might as well use the 'official' I guess :)